ledger/participant-integration-api/src/test/lib/scala/platform/store/backend/StorageBackendTestsIngestion.scala

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.store.backend
+
+import org.scalatest.flatspec.AsyncFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+private[backend] trait StorageBackendTestsIngestion extends Matchers with StorageBackendSpec {
```

Comment:
I think we need something more here. The initializeIngestion needs to be tested exhaustively:
* return value, corner case empty db
* for all types for ingested entries, the overspill entries should be wiped, but not the ones which are before the last ledger end
hint: for the overspill testing:
* either add capabilities of direct table observatin
* or move the ledger end sufficiently ahead, and test with backend retrival methods

Reply:
Added two tests:
- One that checks the return value of `initializeIngestion`
- One that checks `initializeIngestion` deleted overspill entries (by advancing the ledger end)